cura/Arranging/Arrange.py

```diff
@@ -104,15 +111,15 @@ def findNodePlacement(self, node, offset_shape_arr, hull_shape_arr, step = 1):
     def centerFirst(self):
         # Square distance: creates a more round shape
         self._priority = numpy.fromfunction(
-            lambda i, j: (self._offset_x - i) ** 2 + (self._offset_y - j) ** 2, self.shape, dtype=numpy.int32)
+            lambda j, i: (self._offset_x - i) ** 2 + (self._offset_y - j) ** 2, self._shape, dtype=numpy.int32)
```

Comment:
Why flip `i` and `j`?

Reply:
In some places the _priority and _occupied were used as (x, y) and in other places it was (y, x). I found the error when I made the Arranger arrange on a rectangular raster. I use (y, x) everywhere now (in the Arranger) because of a few reasons:
- If you print the table, it will have a good orientation for you, this is the main reason
- This is more theoretical and it has become a habit for some (big) tables. Having "y" as rows makes you calculate the memory location of an item as `y*row_size+x`. If you loop through it from "top left to right, then top to bottom" it may be faster than the other way around. For (x, y) loops over these tables I would place the y loop on the outside for this reason.